rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/importexport/ImportExportController.java

```diff
@@ -198,4 +176,45 @@ public ResponseEntity<String> handleGlobalException(Exception e) {
         LOGGER.error("Unhandled exception: {}", e.getMessage(), e);
         return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(e.getMessage());
     }
+
+    @Operation(summary = "Upload component csv file.", description = "Upload component csv file.", tags = {
+            "Component" })
```

Comment:
The tag should be `ImportExport` as with other endpoints.

Reply:
Also, please document the header needed as done in `downloadComponent()` for the `Content-Type` header (put in `consumes`).